backend-models/service-tests/accessedservice-tests.txt

```diff
@@ -0,0 +1,129 @@
+
+accessedservice.php
+
+-------------------
+
+********
+* pre-req
+********
+
+(checklogin() is TRUE && (hasAccess($userid, $cid, 'w') || isSuperUser($userid,) is TRUE))
+
+--------------------
+
+Test 1: try to update a created user by changing the username
+Row: 62 - 80 and 220 - 238
+Send {
+   $opt = "ADDUSER"
+   $uid = "UNK"
+   $regstatus = "UNK"
+}
+To make sure the update of a user works and what response it will get form the database.
```

Comment:
as of yesterday we no longer compare the results. we send in exampledata to execute a service, and collect the output generated by the service. 

this is the information relevant in this file.
![image](https://user-images.githubusercontent.com/102600391/234267898-a2b16b5a-8893-4392-8c65-6af6d96fd1ac.png)